src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--mon calendar__month-length--31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```

Reply:
`start-day` and `month-length` are modifier names, not block elements

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/208290033-c1401596-ade3-4b7f-9ce4-83a3eae15984.png)